src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```diff
@@ -70,7 +83,34 @@ def get_config():
 
 def get_cover_file_path():
   """Return location of coverage file for Syzkaller."""
-  return os.path.join(get_work_dir(), 'coverfile')
+  file_path = os.path.join(get_work_dir(), 'coverfile')
+  if not os.path.isfile(file_path):
+    open(file_path, 'w+').close()
```

Comment:
why do we need this, when we write to it later anyway?

Reply:
hm I think this was because of the way I implemented it earlier. will fix.